data/decorations/halloween_decorations.json

```diff
@@ -0,0 +1,603 @@
+{
+    "objects": [
```

Comment:
Nitty detail: Can you please resolve the tabbing here, much like the other json files.
Your code editor might support an autoformat option :)

Reply:
Fixed :)

Reply:
I think you forgot to push :(

Reply:
I mean, I think I looked with my ass